src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -395,3 +395,36 @@ export const toggleRegionEditorActive = createAction(
 export const toggleRegionFieldActive = createAction(
   'browser/toggle-region-field-active'
 )<boolean>();
+
+export const clearActiveGenomeId = createAction(
+  'browser/clear-active-genome-id'
+)();
+
+export const clearActiveEnsObjectIdsForGenome = createAction(
+  'browser/clear-active-ens-object-ids-for-genome'
+)<string>();
```

Comment:
Can't we do all that in one single action — something like `deleteGenome` or `removeGenome`? This action would be checked by several reducers, which will need to:
- set active genome id to null if this genome is an active genome;
- remove this genome from `browserEntity.activeEnsObjectIds`
- remove it from the `browserLocation` part of the genome browser state
- remove it from the trackPanel state

We don't need to spawn multiple actions for this purpose.

Reply:
That's done